### ISSUE TITLE
micronaut error loop on docker-compose depoloyment

### ISSUE THREAD

**Issue by avnav88** [airbytehq/airbyte#26390]
### What method are you using to run Airbyte?

Docker

### Platform Version or Helm Chart Version

0.44.5

### What step the error happened?

On deploy

### Revelant information

i followed the guide in terms of pulling from github and running the `run-ab-platform.sh` script.

the only difference is that i am using podman instead of docker.

it looks like micronaut goes on an error loop.

the error impacts the proxy being able to communicate with the server container.

### Relevant log output

```shell
___    _      __          __
   /   |  (_)____/ /_  __  __/ /____
  / /| | / / ___/ __ \/ / / / __/ _ \
 / ___ |/ / /  / /_/ / /_/ / /_/  __/
/_/  |_/_/_/  /_.___/\__, /\__/\___/
                    /____/
        : airbyte-server :
--------------------------------------------------
 Will soon be available at http://localhost:8000/
--------------------------------------------------

  Micronaut (v3.9.1)

2023-05-22 21:49:39 INFO i.m.c.e.DefaultEnvironment(<init>):171 - Established active environments: [ec2, cloud, control-plane]
2023-05-22 21:49:40 INFO i.a.f.ConfigFileClient(<init>):94 - path /flags does not exist, will return default flag values
2023-05-22 21:49:41 INFO i.a.c.t.TemporalUtils(getTemporalClientWhenConnected):248 - Waiting for temporal server...
2023-05-22 21:49:41 WARN i.a.c.t.TemporalUtils(getTemporalClientWhenConnected):259 - Waiting for namespace default to be initialized in temporal...
2023-05-22 21:49:43 INFO i.t.s.WorkflowServiceStubsImpl(<init>):86 - Created WorkflowServiceStubs for channel: ManagedChannelOrphanWrapper{delegate=ManagedChannelImpl{logId=1, target=airbyte-temporal:7233}}
2023-05-22 21:49:49 INFO i.a.c.t.TemporalUtils(getTemporalClientWhenConnected):276 - Temporal namespace default initialized!
2023-05-22 21:49:49 ERROR i.m.r.Micronaut(handleStartupException):338 - Error starting Micronaut server: Bean definition [io.airbyte.server.apis.SourceDefinitionApiController] could not be loaded: Error instantiating bean of type  [io.airbyte.persistence.job.tracker.JobTracker]

Path Taken: new SourceDefinitionApiController(SourceDefinitionsHandler sourceDefinitionsHandler) --> new SourceDefinitionApiController([SourceDefinitionsHandler sourceDefinitionsHandler]) --> new SourceDefinitionsHandler(ConfigRepository configRepository,Supplier uuidSupplier,[SynchronousSchedulerClient schedulerSynchronousClient],AirbyteRemoteOssCatalog remoteOssCatalog,SourceHandler sourceHandler,AirbyteProtocolVersionRange protocolVersionRange) --> SynchronousSchedulerClient.synchronousSchedulerClient(TemporalClient temporalClient,[JobTracker jobTracker],JobErrorReporter jobErrorReporter,OAuthConfigSupplier oAuthConfigSupplier,RouterService routerService,ConfigInjector configInjector) --> JobTracker.jobTracker(ConfigRepository configRepository,JobPersistence jobPersistence,[TrackingClient trackingClient],ActorDefinitionVersionHelper actorDefinitionVersionHelper)
io.micronaut.context.exceptions.BeanInstantiationException: Bean definition [io.airbyte.server.apis.SourceDefinitionApiController] could not be loaded: Error instantiating bean of type  [io.airbyte.persistence.job.tracker.JobTracker]

Path Taken: new SourceDefinitionApiController(SourceDefinitionsHandler sourceDefinitionsHandler) --> new SourceDefinitionApiController([SourceDefinitionsHandler sourceDefinitionsHandler]) --> new SourceDefinitionsHandler(ConfigRepository configRepository,Supplier uuidSupplier,[SynchronousSchedulerClient schedulerSynchronousClient],AirbyteRemoteOssCatalog remoteOssCatalog,SourceHandler sourceHandler,AirbyteProtocolVersionRange protocolVersionRange) --> SynchronousSchedulerClient.synchronousSchedulerClient(TemporalClient temporalClient,[JobTracker jobTracker],JobErrorReporter jobErrorReporter,OAuthConfigSupplier oAuthConfigSupplier,RouterService routerService,ConfigInjector configInjector) --> JobTracker.jobTracker(ConfigRepository configRepository,JobPersistence jobPersistence,[TrackingClient trackingClient],ActorDefinitionVersionHelper actorDefinitionVersionHelper)
        at io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1921) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.context.DefaultApplicationContext.initializeContext(DefaultApplicationContext.java:249) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.context.DefaultBeanContext.readAllBeanDefinitionClasses(DefaultBeanContext.java:3326) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.context.DefaultBeanContext.finalizeConfiguration(DefaultBeanContext.java:3684) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.context.DefaultBeanContext.start(DefaultBeanContext.java:341) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.context.DefaultApplicationContext.start(DefaultApplicationContext.java:194) ~[micronaut-inject-3.9.1.jar:3.9.1]
        at io.micronaut.runtime.Micronaut.start(Micronaut.java:75) ~[micronaut-context-3.9.1.jar:3.9.1]
        at io.micronaut.runtime.Micronaut.run(Micronaut.java:323) ~[micronaut-context-3.9.1.jar:3.9.1]
        at io.micronaut.runtime.Micronaut.run(Micronaut.java:309) ~[micronaut-context-3.9.1.jar:3.9.1]
        at io.airbyte.server.Application.main(Application.java:15) ~[io.airbyte-airbyte-server-0.44.5.jar:?]
Caused by: io.micronaut.context.exceptions.BeanInstantiationException: Error instantiating bean of type  [io.airbyte.persistence.job.tracker.JobTracker]
```

basically get a million of these:
```
2023-05-22 22:42:46 ERROR i.m.r.Micronaut(handleStartupException):338 - Error starting Micronaut server:
Bean definition [io.airbyte.server.apis.SourceDefinitionApiController] could not be loaded:
Error instantiating bean of type  [io.airbyte.persistence.job.tracker.JobTracker]
```

all the containers are running however, including the server container.
```

**Comment by avnav88** [comment]
after some looking around and testing, i'm pretty sure it has to do with the following envs:
```
WORKSPACE_ROOT=???
WORKSPACE_DOCKER_MOUNT=airbyte_workspace
```

I added some of these, but they don't fix the issue:

```

S3_LOG_BUCKET=airbyte-logs
S3_LOG_BUCKET_REGION=
S3_MINIO_ENDPOINT=http://10.235.77.22:9002
S3_PATH_STYLE_ACCESS=true

WORKER_LOGS_STORAGE_TYPE=MINIO
WORKER_STATE_STORAGE_TYPE=MINIO

AWS_ACCESS_KEY_ID=admin
AWS_SECRET_ACCESS_KEY=admin123
S3_LOG_BUCKET_REGION=

```

when i commented out the first two, airbyte actually started loading the webpage, but then crapped out.

**Comment by mfsiega-airbyte** [comment]
Podman is not officially supported. If this also occurring on Docker feel free to re-open, or also welcome PRs to add Podman support.
